packages/webpack/src/buildtime/policyGenerator.js

```diff
@@ -133,6 +196,12 @@ module.exports = {
     }
 
     const policy = moduleInspector.generatePolicy({})
+    Object.entries(meta).forEach(([packageName, packageMeta]) => {
+      if (!policy.resources[packageName]) {
+        return
+      }
+      policy.resources[packageName].meta = packageMeta
+    })
```

Comment:
What's this doing?  I'm not aware of any other place we're actually mutating a policy